website/views.py

```diff
@@ -1,14 +1,35 @@
 from django.shortcuts import render,redirect
 from django.contrib.auth import authenticate,login,logout
 from django.contrib import messages
-from .forms import SignUpForm,AddRecordForm
-from .models import Record
+from website.forms import SignUpForm,RecordForm
+from website.models import Record
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views import View
 
-# Create your views here.
-def home(request):
-    records=Record.objects.all()
+# Function Based Home View
```

Comment:
It would be better if you make a copy of this file something like `function_based_views.py` and copy all function based view in that so have a backup of these for future reference.